fenv_lmi.hpp

```diff
@@ -66,10 +46,10 @@
 /// with a valid argument, and invalid arguments can be disallowed by
 /// C++'s type system).
 ///
+/// When using x87, i.e. the symbol LMI_X87 is defined, the following
+/// syntactically-similar functions are also available:
 ///   e_ieee754_precision fenv_precision();
 ///   void fenv_precision(e_ieee754_precision);
-/// The precision functions similarly resemble GNU/Linux functions
-/// fe[gs]etprecision().
```

Comment:
I removed the mention of these functions because they don't seem to actually exist.